policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+* Share nominations publicly
+    * Each team representative must publicly share their team's nominations at least ten days before the final election meeting. This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting
+
+### Election Meeting
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will say who they nominate now and share reasons for their change if that applies.
+* Facilitator proposes a candidate:
+    * Based on gathered data, the facilitator suggests a candidate they believe will receive the group's consent.
+* Seek objections to the facilitator's proposal
+* Finalize the selection:
+    * Discuss and attempt to resolve any objections, if possible.
+    * Consent has been reached if the proposed choice has no remaining objections! :tada:
+    * If objections persist and the current proposal reaches an impasse, the facilitator should make a new proposal based on the discussion and feedback received.
+    * Continue seeking consensus and addressing objections until a satisfactory candidate is selected.
+        * Note that this process seeks to produce a candidate everyone is satisfied with and has no objections to. Avoid lengthy debates over preferences not related to objections or unmet requirements.
+* Document and announce the selection
+
```

Comment:
Do we need a stalemate contingency plan? I get that we foresee that being highly unlikely especially in a high trust environment, and I'm absolutely fine with it only being a last resort, but it's not impossible and I'd rather have some sort of process that is 100% guaranteed to produce a result instead of *gestures to a governance crisis*.

Reply:
I don't feel like we're risking a governance crisis if we don't select new project directors quickly enough, we'll just extend our terms again (narrator voice: I do not want this). I'm not opposed to the idea of a stalemate contingency plan, so feel free to suggest a plan if you have one in mind, but I don't feel like this is something we will need, and if we do need it, I figure we will still be able to add it later without coming anywhere near a crisis.

Reply:
I do worry that we might be in a situation where we cannot reach a consensus on a set of candidates and that there are hard objections to enough candidates as to make it impossible to fill a seat. What happens then?

Reply:
> I do worry that we might be in a situation where we cannot reach a consensus on a set of candidates and that there are hard objections to enough candidates as to make it impossible to fill a seat. What happens then?

I think we'd just keep trying or come up with an alternative approach if this one proves ineffective; that said, I feel that's unlikely to occur, given how many ways we have to help us reach a consensus[^1]. I'd rather encounter such a scenario and come up with solutions based on concrete experience rather than trying to defend against hypothetical concerns.

[^1]: What are those objections? Are they based on preference, or are they based on the role description and the established shared goals? Are they resolvable? Can we measure the concern? Can we shorten the term? etc, etc, etc

Reply:
@rylev - Do you have any more thoughts here?

I think this not getting a consensus is unlikely enough that I'm fine with not stating anything, but it may not hurt to say something since we're talking about it.

I can think of a couple of stalemate contingency plans:

1. The board votes to extend terms and we repeat the process from scratch. We probably need to say something like "none of the candidates put forward were acceptable, so choose all new people when putting forward nominations." I'm not sure what to do if say we needed three people, we were able to reach consensus on two, but there was no acceptable candidate for the third. Maybe we keep the two we agreed on and re-run the selection process for just the one open seat? At any rate, I think if we go down this path, it's likely we won't solicit enough nominations to be successful the second time around.
2. The Council puts forward their own nominations that would be acceptable. This probably keeps things more time efficient and hopefully avoids repeating the stalemate, but it's also less significant. If we go this path, when we announce the results we'd definitely owe some kind of explanation for why the person we selected was not in the original set. I guess this route also doesn't give the project a chance to raise objections for the new candidates the council nominates on their own.
3. Just say "if a stalemate occurs, the council will create another Project Director Election Committee to revise the process so we can try again." We'd probably also vote to extend terms here as well.

If we want to say anything, I like option 3. It keeps us from having undefined behavior in the process, but it also means we don't have to attempt to solve unknown problems without sufficient information just yet.

Reply:
I'll weigh in and say that my worry is much more about not finding enough people willing and able (time commitment) to do this work -- rather than us not being able to agree to a slate of nominees. Even with a partial rotation every ~1-2 years, we still end up needing to pick a pretty large number of people and a group with harsher restrictions than the council on corporate affiliation (IIRC, no more than 1, assuming it's for a company with a member on the board directly).

I'm personally happy with this proposal as the first attempt and I think *if* there is a contingency of some kind needed, I don't know that we gain much by trying to decide on it now vs. after we need it. If anything, I might add an explicit disclaimer to any documentation we put out about this that we're trying something new here and we'll re-evaluate depending on the results how much people liked the process and whether it was successful.

Reply:
I added some text in 2984a52c0f5cc19264f7e52e4114ed87d72cdda0 that hopefully resolves this. Please mark the conversation as resolved if you're happy with it.